src/renderer/src/views/Home/Settings/ProjectConfig/MemberManagmentModal/DeviceInfoContent/DeviceInfo.tsx

```diff
@@ -0,0 +1,193 @@
+            </Column>
+          </Row>
+          <Row>
+            <Column>
+              <Row>
+                <Typography fontSize={18}>{t(m.role)}</Typography>
+              </Row>
+              {changingRole ? (
+                <Row alignItems="flex-start">
+                  <Select value={device.role} onChange={handleSelectChange}>
+                    <MenuItem
+                      onClick={() => {
+                        setChangingRole(false)
+                      }}
+                      value={'coordinator'}
+                    >
+                      {' '}
+                      {t(m.coordinator)}{' '}
+                    </MenuItem>
+                    <MenuItem
+                      onClick={() => {
+                        setChangingRole(false)
+                      }}
+                      value={'participant'}
+                    >
+                      {' '}
+                      {t(m.participant)}
+                    </MenuItem>
+                  </Select>
+                </Row>
+              ) : (
+                <Row alignItems="center">
+                  <Column>
+                    {device.role === 'coordinator' ? (
+                      <ManageAccountsIcon fontSize="large" htmlColor={theme.black} />
+                    ) : (
+                      <PersonAddIcon fontSize="large" htmlColor={theme.black} />
+                    )}
+                  </Column>
+                  <Column marginLeft={1}>
+                    <Typography textTransform={'capitalize'} fontSize={16}>
+                      {device.role}
+                    </Typography>
+                  </Column>
+                  <Column>
+                    <Button
+                      noBorder
+                      variant="text"
+                      onClick={() => setChangingRole(true)}
+                      style={{ marginLeft: 10 }}
+                    >
+                      <Typography style={{ textTransform: 'none' }} fontSize={14}>
+                        {t(m.changeRole)}
+                      </Typography>
+                    </Button>
+                  </Column>
+                </Row>
+              )}
+            </Column>
+          </Row>
+          <Row>
+            <Column>
+              <Typography fontSize={18}>{t(m.pastDeviceNames)}</Typography>
+              {device.pastDeviceNames?.map((name) => (
+                <Row>
+                  <Typography fontSize={18}>{name}</Typography>
+                </Row>
+              ))}
+            </Column>
+          </Row>
+        </Column>
+      </Row>
+      <Row
+        justifyContent="space-between"
+        alignItems="center"
+        sx={{
+          paddingY: 2,
+          paddingX: 4,
+          borderTop: '1px solid',
+          borderColor: theme.grey.light,
+          width: '100%',
+        }}
+      >
+        <Button noBorder variant="text" sx={{ color: theme.warningRed }} onClick={moveToConfirmationScreen}>
```

Comment:
similar comment about using `color="warning"` here to get the proper hover state color

before (on hover):

<img width="186" alt="image" src="https://user-images.githubusercontent.com/18542095/234897409-ec347542-fd48-4cfe-8449-aa07fe00ceab.png">

after (on hover):

<img width="184" alt="image" src="https://user-images.githubusercontent.com/18542095/234897257-2c27287c-7b3a-4275-9bbb-509759775b5d.png">